ironfish-cli/src/commands/wallet/balances.ts

```diff
@@ -82,6 +89,15 @@ export class BalancesCommand extends IronfishCommand {
       }
     }
 
-    CliUx.ux.table(response.content.balances, columns)
+    response.content.balances.sort((left, right) =>
+      compareAssets(
+        left.assetName,
+        left.assetVerification,
+        right.assetName,
+        right.assetVerification,
+      ),
+    )
+
+    CliUx.ux.table(response.content.balances, columns, flags)
```

Comment:
Adding `flags` here is not strictly needed for the goals of this PR. This is to add support for `--output=csv|json|yaml`, as well as other options.